src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -108,7 +111,11 @@ def location(self, dirname: str) -> None:
         else:
             raise FileNotFoundError(f"given path {dirname} is not valid")
 
-    def init_from_file(self) -> None:
+    def init_from_file(
+        self, 
+        config_type: str, 
```

Comment:
it would seem like this config_type is never used.
is it really needed?

And if we are removing it from here, then maybe we remove it from the `get_profile` method as well?

Reply:
Yes, that makes sense :blush: